### PR TITLE
Adds "Google Authenticator" to the list of recommended extensions

### PR DESCRIPTION
**Pull request by Kubik-Rubik**
This PR adds the extension "Google Authenticator" to the "System" list where recommended extensions are listed.

Install the patch and check the functionality! :-)

screenshot 2016-08-15 12 47 00

screenshot 2016-08-15 12 47 17

### CODE REVIEW

**Comment by sibprogrammer** [comment]
You need to increase the version number in meta.xml file.

**Comment by xgin** [comment]
- [x] Add promo block (see src/plib/library/Promo/Home.php for details)

File: src/plib/controllers/IndexController.php [hunk 1]
```diff
@@ -171,6 +171,8 @@ public function systemAction()
                 Modules_SecurityAdvisor_Datagrid::install();
             } elseif ($this->_getParam('btn_patchman_install')) {
                 Modules_SecurityAdvisor_Patchman::install();
+            } elseif ($this->_getParam('btn_googleauthenticator_install')) {
+	            Modules_SecurityAdvisor_GoogleAuthenticator::install();
```
Comment (vlikhtanskiy) [inline src/plib/controllers/IndexController.php]: Use white spaces instead of tabs.

File: src/plib/library/GoogleAuthenticator.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?php
```
Comment (vlikhtanskiy) [inline src/plib/library/GoogleAuthenticator.php:1]: Use everywhere white spaces instead of tabs.

File: src/plib/resources/locales/en-US.php [hunk 3]
```diff
@@ -69,6 +69,10 @@
     'controllers.system.patchman' => 'Patchman',
     'controllers.system.patchmanActivate' => 'Activate Patchman',
     'controllers.system.patchmanInstall' => 'Install Patchman',
+    'controllers.system.googleauthenticatorDesc' => 'Google Authenticator add a 2 Factor Authentication (2FA) to the login of your Plesk instance.',
```
Comment (vlikhtanskiy) [inline src/plib/resources/locales/en-US.php]: "adds"

**Comment by Kubik-Rubik** [comment]
Solved all issues, please re-check. Thank you!

File: src/plib/controllers/IndexController.php (hunk 1 above)
Comment (vlikhtanskiy) [inline src/plib/controllers/IndexController.php]: Still tab in the beginning of the line.

File: src/plib/library/Promo/Home.php [hunk 4]
```diff
@@ -29,6 +29,7 @@ public function getButtonUrl()
             case 'http2' :
             case 'datagrid' :
             case 'patchman' :
+            case 'google-authenticator' :
```
Comment (vlikhtanskiy) [inline src/plib/library/Promo/Home.php]: You haven't added locale key for this step so possibly promo is corrupted here.

Comment (Kubik-Rubik) [inline src/plib/library/Promo/Home.php]: Done! Thanks.

**Pull request merged**